packages/tables/src/elements/Row.spec.tsx

```diff
@@ -176,4 +176,35 @@ describe('Row', () => {
       expect(getByTestId('row')).toHaveStyleRule('height', '64px');
     });
   });
+
+  describe('ReadOnly', () => {
```

Comment:
Would it be worth it to have tests asserting that rows _are_ interactable when the read-only prop isn't applied? Since that's all guarded by logic now?

Reply:
That was going to be the next step once I was able to target the styles successfully. Looks like that isn't going to be possible unless we target the `StyledRow` directly. Going to remove this so we can get a passing deployment.